modules/cluster-logging-upgrading-logging-from-44.adoc

```diff
@@ -0,0 +1,304 @@
++
+[source,terminal]
+----
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    retentionPolicy:
+      application:
+        maxAge: 7d
+      infra:
+        maxAge: 1h
+      audit:
+        maxAge: 1h
+...
+----
+
+.. Click *Save*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+The Cluster Logging Operator is shown as 4.6. For example:
++
+[source,terminal]
+----
+Cluster Logging
+4.6.0-202007012112.p0 provided
+by Red Hat, Inc
+----
++
+Wait for the *Status* field to report *Succeeded*.
+
+. Update the Elasticsearch Operator:
+
+.. From the OpenShift web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-operators-redhat` project.
+
+.. If the Elasticsearch Operator is installed on the `openshift-logging` namespace, uninstall the Elasticsearch Operator.
++
+[NOTE]
+====
+This uninstall action does _not_ delete the existing data, volumes, and so on.
+====
+
+.. Click *Operators* -> *OperatorHub*. Search for and select `OpenShift Elasticsearch Operator`.
+
+ .. Click *Install*
+ +
+ The *Install operator* page opens.
+
+.. For *Update Channel*, make sure *4.6* is selected.
+
+.. For *Installed Namepsace*, select `openshift-operators-redhat` namespace.
+
+.. Click *Install*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+The Elasticsearch Operator is shown as 4.6. For example:
++
+[source,terminal]
+----
+Elasticsearch Operator
+4.6.0-202007012112.p0 provided
+by Red Hat, Inc
+----
++
+Wait for the *Status* field to report *Succeeded*.
+
+. Check the logging components:
+
+.. Ensure that all Elasticsearch pods are in the *Ready* status:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-logging --selector component=elasticsearch
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY   STATUS    RESTARTS   AGE
+elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk   2/2     Running   0          31m
+elasticsearch-cdm-1pbrl44l-2-5c6d87589f-gx5hk   2/2     Running   0          30m
+elasticsearch-cdm-1pbrl44l-3-88df5d47-m45jc     2/2     Running   0          29m
+----
++
+.. Ensure that the Elasticsearch cluster is healthy:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk -- es_cluster_health
```

Comment:
There might be an extra space between the dashes and the option here:     -- es_cluster_health
If the space needs to deleted, the OCP doc should probably change too.

Reply:
I'm not sure. Sometimes, with Github commands anyway, the `--` followed by a space is needed for certain actions. Let's see what QE says - especially since this was the originally approved version from OCP.

Reply:
Engineering says that the space is correct as is.